packages/pyright-internal/src/localization/package.nls.ru.json

```diff
@@ -505,10 +522,34 @@
         "typeCallNotAllowed": "Вызов type() не разрешен в аннотации типа",
         "typeCheckOnly": "\"{name}\" помечено как @type_check_only и может использоваться только в аннотациях типа",
         "typeCommentDeprecated": "Комментарии типа не рекомендуются, используйте аннотации типа",
+        "typedDictAccess": "Нельзя получить элемент из TypedDict",
+        "typedDictAssignedName": "TypedDict необходимо присвоить переменной с именем \"{name}\"",
+        "typedDictBadVar": "Классы TypedDict могут содержать только аннотации типа",
+        "typedDictBaseClass": "Все базовые классы для классов TypedDict также должны быть классами TypedDict",
+        "typedDictBoolParam": "Для параметра \"{name}\" ожидается значение True или False",
+        "typedDictClosedExtras": "Базовый класс \"{name}\" представляет собой closed TypedDict; дополнительные элементы должны иметь тип \"{type}\"",
+        "typedDictClosedNoExtras": "Базовый класс \"{name}\" представляет собой closed TypedDict; дополнительные элементы не разрешены",
+        "typedDictDelete": "Нельзя удалить элемент из TypedDict",
+        "typedDictEmptyName": "Имена в TypedDict не могут быть пустыми",
+        "typedDictEntryName": "Для имени записи словаря ожидается строковый литерал",
+        "typedDictEntryUnique": "Имена в словаре должны быть уникальными",
+        "typedDictExtraArgs": "Дополнительные аргументы TypedDict не поддерживаются",
+        "typedDictExtraItemsClosed": "TypedDict не может быть \"closed\", если он поддерживает дополнительные элементы",
+        "typedDictFieldNotRequiredRedefinition": "Элемент TypedDict \"{name}\" нельзя переопределить как NotRequired",
+        "typedDictFieldReadOnlyRedefinition": "Элемент TypedDict \"{name}\" нельзя переопределить как доступный ReadOnly",
+        "typedDictFieldRequiredRedefinition": "Элемент TypedDict \"{name}\" нельзя переопределить как Required",
+        "typedDictFirstArg": "В качестве первого аргумента ожидается имя класса TypedDict",
+        "typedDictInClassPattern": "Класс TypedDict не разрешен в шаблоне класса",
+        "typedDictInitsubclassParameter": "TypedDict не поддерживает параметр __init_subclass__ \"{name}\"",
+        "typedDictNotAllowed": "Невозможно использовать \"TypedDict\" в этом контексте",
+        "typedDictSecondArgDict": "После имени передайте в TypedDict либо словарь, либо именованные аргумента",
+        "typedDictSecondArgDictEntry": "Ожидается простая запись словаря",
+        "typedDictSet": "Нельзя назначить элемент в TypedDict",
```

Comment:
these were moved with no changes?

Reply:
I sorted the names alphabetically, because for some reason they were in the wrong order. Should I revert?

Reply:
i think so, otherwise when i merge upstream i might accidentally duplicate them in their original spots if they change

Reply:
I reverted that change, and added basedpyright-specific items to the end to match the English localizations. I don't know if there's any point in sorting them, I search individual items with Ctrl+F anyway